scripts/specs/publishToNpm.spec.js

```diff
@@ -34,7 +34,7 @@ describe("publishToNpm", () => {
     );
     expect(logger.warn).not.toHaveBeenCalled();
     expect(logger.info).toHaveBeenCalledWith("Publishing NPM package.");
-    expect(exec).toHaveBeenCalledOnceWith("npm publish", jasmine.any(String));
+    expect(exec).toHaveBeenCalledWith("npm publish", jasmine.any(String));
```

Comment:
On my machine, this test was failing because the spy was also capturing a `sleep` command. Maybe somebody can try to run this test on their machine using `toHaveBeenCalledOnceWith` and see if they see any errors. I have node 22 and NPM 10.8.1 on my machine.